docs-src/docs/annotation-integration/review.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Review
+---
+
+A review is a decision that determines whether an [annotation](../#annotation) is of sufficient quality or if it needs 
+improvement. In order to facilitate efficient improvements, the person or system that made the decision can provide 
+[feedback](#feedback) data that details potential errors or mistakes.
+
+## Posting a review
+
+The current integration capabilities only allow posting reviews for delivered annotations. The annotations are 
+identified using their open label UUID that can be found in the metadata (and in the file name) of [downloaded 
+annotations](../kognic-io/annotations).
+
+When posting a review, the API expects feedback data as well as a boolean `accepted` that indicates whether the 
+quality is perceived as sufficient or not. The API also expects an enum `workflow` that details how the improvement 
+should happen. For a rejected review and the workflow 'correct', a single correction task is triggered, with no 
+follow-up Review task. The completed Correction task becomes the new delivered Annotation. 
+
+:::caution
+This integration API is currently limited to rejecting delivered annotations, the possible values of these input 
+parameters are thus limited to `accepted=false` and `workflow="correct"`. They are included in the interface at this
+stage in order to allow future expansions on the available use cases, workflows and actions.
+:::
+
+A successful review post will return a UUID that identifies the review, and that can be used to fetch the posted
+feedback data. This will also create a task according to the chosen workflow. Depending on project set up, there might 
+be a need for this to be coordinated with the project's managers.
+
+## Feedback
+
+A feedback item details something noteworthy in an annotation. In order to do this the following information is 
```

Comment:
Would it help to also point to the user docs to show the "UI side" of the same thing? (If it is the same thing, that is? Idk)
Like https://docs.kognic.com/giving-feedback

Reply:
Great point, I'll refer to those docs from the top section of this page, thanks!